.env

```
@@ -1,4 +1,4 @@
-NEXT_PUBLIC_SITE_NAME = 开放黑客松
+NEXT_PUBLIC_SITE_NAME = 黑客松开放平台
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)